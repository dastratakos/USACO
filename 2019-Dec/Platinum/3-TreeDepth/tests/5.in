18 117 592996081
