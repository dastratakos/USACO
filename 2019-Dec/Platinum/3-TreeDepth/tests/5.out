547877949 145016137 11583074 378068048 117925164 107498081 166172066 12316002 551291515 285969093 566307198 348339128 63253558 210555336 109483805 107852039 2891403 164502295
